### PR TITLE
libbladerf 2019.07

### PR DESCRIPTION
**Pull request by sielicki**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by sielicki** [comment]
I'm at somewhat of a loss for what to do here; it looks to me like upstream doesn't actually support builds from their release tarballs.

https://github.com/Nuand/bladeRF/blob/master/host/common/thirdparty/ad936x/CMakeLists.txt#L120-L144

Probably need to point this at git instead? First time contributing to homebrew formulas, thanks for the hand holding in advance.

**Comment by SMillerDev** [comment]
It seems to have supported it before. Try and build it locally to see if you can make more sense of the errors there.

**Comment by sielicki** [comment]
No, see this commit (landing after 2018.8, the current formula version)

https://github.com/Nuand/bladeRF/commit/48ec0a40a65727b8007fcbd94965ff201cd4203b#diff-aebf5aeb51af813b47effc97b01dfe50

Upstream has effectively dropped support of release tarballs (as github release tarballs do not include submodule code)

**Comment by SMillerDev** [comment]
In that case that seems like the way to go.

**Comment by sielicki** [comment]
Sorry for the laziness here, but I looked over the Formula Cookbook and it's not immediately apparent how I can push a change that would build a tagged release from git rather than from a tarball. Do I just remove the URL to the tarball and do something like, 

```
  head "https://github.com/some/package.git", :revision => "090930930295adslfknsdfsdaffnasd13"
```

Again, thank you for the excessive hand holding.

**Comment by SMillerDev** [comment]
Yeah, that's mostly it. But you'd need to specify a tag and a revision as a policy. (You can `git grep ":revision"` to find examples)

**Comment by sielicki** [comment]
Can you squash-rebase those fixups on merge, or do you need me to force push?

Thanks for the help, I think it's set now.

File: Formula/libbladerf.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-  url "https://github.com/Nuand/bladeRF/archive/2018.08.tar.gz"
-  sha256 "6288c230dad26e32236a4b60f0b14c129e6fa0ad91bcf1c40abe8789b352e51f"
   head "https://github.com/Nuand/bladeRF.git"
 
+  stable do
+    url "https://github.com/Nuand/bladeRF.git",
+        :tag      => "2019.07",
+        :revision => "991bba2f9c4d000f000077cc465878d303417e26"
+  end
+
```
Comment (SMillerDev) [inline Formula/libbladerf.rb:11]: Please don't move this into a stable block but keep it in the same place it was before.

**Review by SMillerDev — APPROVED** [review]
Thanks @sielicki ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**